app/src/main/java/org/cru/godtools/ui/tools/analytics/model/ToolsAnalyticsActionEvent.kt

```diff
@@ -0,0 +1,14 @@
+package org.cru.godtools.ui.tools.analytics.model
+
+import org.cru.godtools.analytics.model.AnalyticsActionEvent
+import org.cru.godtools.analytics.model.AnalyticsSystem
+
+sealed class ToolsAnalyticsActionEvent(action: String) : AnalyticsActionEvent(action) {
+    override fun isForSystem(system: AnalyticsSystem): Boolean {
+        return system == AnalyticsSystem.ADOBE
+    }
```

Comment:
this is a very simple function and is a good candidate to follow the single expression function idiom. https://kotlinlang.org/docs/reference/idioms.html#single-expression-functions